llama_stack/providers/remote/inference/tgi/config.py

```diff
@@ -12,19 +12,20 @@
 
 @json_schema_type
 class TGIImplConfig(BaseModel):
-    host: str = "localhost"
-    port: int = 8080
-    protocol: str = "http"
-
-    @property
-    def url(self) -> str:
-        return f"{self.protocol}://{self.host}:{self.port}"
-
+    url: str = Field(
```

Comment:
Do we need to revert back all `<protocol>://<host>:<port>` to `url` for `RemoteProviderConfig`?

Reply:
yeah we should do that I agree. I will audit that next.